packages/contracts/test/factory/dao-factory.ts

```diff
@@ -211,14 +231,14 @@ describe('DAOFactory: ', function () {
       mintAmount
     );
 
-    const EXECUTE_PERMISSION_ID = await managingDao.EXECUTE_PERMISSION_ID();
+    const EXECUTE_PERMISSION_ID = await createdDao.EXECUTE_PERMISSION_ID();
 
     const DAOPermissions = await Promise.all([
-      managingDao.SET_METADATA_PERMISSION_ID(),
-      managingDao.ROOT_PERMISSION_ID(),
-      managingDao.WITHDRAW_PERMISSION_ID(),
-      managingDao.UPGRADE_PERMISSION_ID(),
-      managingDao.SET_SIGNATURE_VALIDATOR_PERMISSION_ID(),
+      createdDao.SET_METADATA_PERMISSION_ID(),
```

Comment:
why not `createdDAO` instead of `createdDao`? so

Reply:
[Some styleguides](https://stackoverflow.com/questions/15526107/acronyms-in-camelcase#:~:text=When%20using%20acronyms%2C%20use%20Pascal,in%20identifiers%20or%20parameter%20names.) suggest to not treat abbreviations/acronyms differently in camel case and argue for not capitalizing them. For the JS files at least, I was under the impression that we are not doing it.